src/components/Calculator.js

```diff
@@ -1,187 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      obj: {},
-    };
-    this.showResult = this.showResult.bind(this);
-  }
+const Calculator = () => {
+  const [obj, setResult] = useState({});
```

Comment:
- [ ] You did great refactoring the class-based Calculator component into a functional component with hooks. But when a number is divided by zero it throws a browser error. Kindly add an error handling method or condition so that the error message is displayed in the input field. 

![Screenshot from 2021-09-16 16-50-51](https://user-images.githubusercontent.com/64894927/133600311-947352bb-2e9c-4fa6-aeba-0aefc3e0c151.png)